elibrary-hibernate/pom.xml

```diff
@@ -40,12 +40,12 @@
 					<propertyFile>liquibase.properties</propertyFile>
 					<changeLogFile>src/main/resources/db-changelog-master.xml</changeLogFile>
 					<!-- comment if running locally -->
-					<url>
```

Comment:
Please remove it as it's a local change. 🗑️

Reply:
I did it.